Sources/Commands/Options.swift

```diff
@@ -103,6 +103,9 @@ public class ToolOptions {
     /// Whether to use the explicit module build flow (with the integrated driver)
     public var useExplicitModuleBuild: Bool = false
 
+    /// Whether to output a graphviz file visualization of the combined job graph for all targets
+    public var printManifestGraphviz: Bool = false
```

Comment:
It would be great if this were phrased as the stream to which to write the GraphViz representation, rather than always writing it to stdout.  That way we could enable it from IDEs such as Xcode, where emitting to stdout isn't always the right thing.  This could be done as a follow-on PR.